src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -100,6 +100,11 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
         }
         val userId = request.patchesList.first().userId
         LOG.info("Get request for create new patch of file={} by user={}", request.fileId, userId)
+        synchronized(this.fileBuffer) {
+            if (this.fileBuffer[request.info.projectId] == null) {
+                this.fileBuffer[request.info.projectId] = ConcurrentHashMap()
```

Comment:
I see that `fileBuffer` is already configured with default value. This requires using `getValue` though, but maybe it worth doing?

Reply:
Now I m using `getOrPut`, cause i want to init project as fast as possible: earlier we were doing it in the end of the method so we had to lock `fileBuffer` for all of it